pds/contracts/IPackNFT.cdc

```diff
@@ -1,98 +1,96 @@
 import Crypto
-import NonFungibleToken from 0x{{.NonFungibleToken}}
+import NonFungibleToken from "NonFungibleToken"
 
+access(all) contract interface IPackNFT{
+
+    /// Entitlement to perform operations on the PackNFT
+    ///
+    access(all) entitlement Operatable
 
-pub contract interface IPackNFT{
     /// StoragePath for Collection Resource
     ///
-    pub let CollectionStoragePath: StoragePath
+    access(all) let CollectionStoragePath: StoragePath
     /// PublicPath expected for deposit
     ///
-    pub let CollectionPublicPath: PublicPath
+    access(all) let CollectionPublicPath: PublicPath
     /// PublicPath for receiving PackNFT
     ///
-    pub let CollectionIPackNFTPublicPath: PublicPath
+    access(all) let CollectionIPackNFTPublicPath: PublicPath
     /// StoragePath for the PackNFT Operator Resource (issuer owns this)
     ///
-    pub let OperatorStoragePath: StoragePath
-    /// PrivatePath to share IOperator interfaces with Operator (typically with PDS account)
-    ///
-    pub let OperatorPrivPath: PrivatePath
+    access(all) let OperatorStoragePath: StoragePath
     /// Request for Reveal
     ///
-    pub event RevealRequest(id: UInt64, openRequest: Bool)
+    access(all) event RevealRequest(id: UInt64, openRequest: Bool)
     /// Request for Open
     ///
     /// This is emitted when owner of a PackNFT request for the entitled NFT to be
     /// deposited to its account
-    pub event OpenRequest(id: UInt64)
+    access(all) event OpenRequest(id: UInt64)
     /// Burned
     ///
     /// Emitted when a PackNFT has been burned
-    pub event Burned(id: UInt64 )
+    access(all) event Burned(id: UInt64 )
     /// Opened
     ///
     /// Emitted when a packNFT has been opened
-    pub event Opened(id: UInt64)
+    access(all) event Opened(id: UInt64)
 
-    pub enum Status: UInt8 {
-        pub case Sealed
-        pub case Revealed
-        pub case Opened
-    }
+    // Enums cannot be declared anymore in interfaces in Cadence 1.0
+    // access(all) enum Status: UInt8 {
```

Comment:
The `Status` enum remain needed in `PackNFT` contracts, but we remove it in `IPackNFT` because concrete type declarations are not allowed in contract interfaces anymore